Header header
geometry_msgs/Pose2D Left
std_msgs/UInt8 State_left
geometry_msgs/Pose2D Right
std_msgs/UInt8 State_right
geometry_msgs/Pose2D Bottom
std_msgs/UInt8 State_bottom